app/models/framework_tender.rb

```diff
@@ -1,21 +1,17 @@
 class FrameworkTender < ApplicationRecord
-     enum status: [:draft, :approved, :canceled, :closed, :archived]
+     enum status: [:open, :closed]
      validates :year, :half_year,:presence => true
      validates_uniqueness_of :year, :scope => :half_year
      has_many :bids
      belongs_to :user, foreign_key: 'certified_by'
 
      def self.get_index(status)
```

Comment:
@nathnael not sure why we're juggling with strings and symbols? What's the purpose of converting strings into symbols?